### PR TITLE
Created a Bug Report Issue

### PR DESCRIPTION
**Pull request by sigrunixia**
So, the repo has been getting a lot of bug requests lately. I know not everyone answers them, but something is better than nothing, right?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+### Calendar Data Collecting
+
+__What Calendar system were you using?__ 
+*Harptos, Earth, Etc* 
+
+__Was it created within the plugin, or imported from Fantasy Calendars online?__
```
Comment (V13Axel) [inline .github/ISSUE_TEMPLATE/bug-report.md:26]: Just a note: The web app's name is Fantasy Calendar, singular. 

Do that, and I think this is fine.

**Review by V13Axel — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (sigrunixia) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Should be updated now. :)

**Review by V13Axel — APPROVED** [review]

**Comment by V13Axel** [comment]
I'm gonna leave it up to @valentine195 for whether anything is missing from the actual questions themselves

**Comment by valentine195** [comment]
LGTM ty

**Pull request merged**